src/App.tsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route, HashRouter } from 'react-router-dom';
+
 import { List } from './modules/oh-indicators-list';
-import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 const App = () => {
   return (
-    <Router>
+    <HashRouter>
```

Comment:
@chooidos check out what's the difference between the default one and this one, keywords: location strategy, hash routing